### PR TITLE
A Beginner's Guide to Open Source - A Course

### PR DESCRIPTION
**Pull request by hollenberry**
## Overview
**TL;DR**
This course resolves an issue discussed internally, which is to create a course that is a beginner's guide to getting involved with Open Source. 

#### Prerequisites
- [ ] Complete efforts in https://github.com/github/training-kit/issues/415

#### Outline

- How to find a relevant repository
  - Anecdotal stories about how anyone can contribute to any Open Source repository
  - Explore 
    - Sites the user frequents
    - Communities 
      - up-for-grabs 
      - Hacktoberfestlikes
    - Labels
    - Search 
- First Steps
     - README.md 
     - Do you know how to use GitHub? Link to On Demand. 
  - Checking if your repository is active 
     - "Health" tracking sites
     - Pulse, graphs
     - Open Issues/PRs
     - Last Opened vs. Last Merged PR
- Contributing
  - Finding/creating the right issues
  - Contributing.md
  - Interacting Well. Pro-tips for Issues and PRs
     - Using Templates
     - Screenshots
     - Merge Strategies
       - Fork and Pull Close Look
- Staying active
  - Communication
    - Notification management
    - Joining communities (Slack, Gitter, forums)  
  - Responses
      - Waiting for a response
      - GitHub.com/pulls
      - Responding to feedback

### Tasks
- [ ] Finish updating Training Kit to act as a strong [example](https://github.com/github/training-kit/issues/415). 
- [x] Add OS Course to Header 
- [x] Add sidebar, pages, and titles
- [ ] Text Content
- [ ] Video Content
- [ ] Gif Content 
- [ ] Test internally
- [ ] Launch near February 13th (concurrent with Open Source Guides)
- [ ] Promote

#### Future initiatives for consideration after this offering is created
- A course on how to be a good maintainer/creating a welcoming Open Source Community (duplicates efforts in Open Source guide).

### Resources
- [Open Source Guide](https://github.com/github/open-source-guide)
- [Zalando Resources](https://github.com/zalando/zalando-howto-open-source#how-to-become-a-contributor)

cc:// @github/services-training

### CODE REVIEW

**Comment by beardofedu** [comment]
@hollenberry Let me know what colors you want me to try and implement for the notification drop-downs 👍

**Comment by hollenberry** [comment]
>  Let me know what colors you want me to try and implement for the notification drop-downs 👍

@beardofedu I threw in red for now! But I'm happy with anything that's a strong contrast to the other options. 😄

**Comment by hollenberry** [comment]
Update:

Hey @github/services-training! Just wanted to bring everyone's attention here. I'm going to continue to work on finishing up the text content through the end of the day (and probably a bit while you're all traveling on Monday). 

As is, I'd love your feedback so far!
- I tried to have a personality, like @beardofedu. Jury's still out on if it feels as charming. 
- I've been working on cross-linking a lot of ✨ resources, I'd love to add more if you can think of any! I think it really fits for this course, especially. 
- Any other feedback/edits/commits! 
- I would be over the 🌔 for any help with .gif's especially

The Open Source team launches their Guides on 2/13. I'm going to grind as much as I can to have us get this course launched in Beta by then as well.

**Comment by stoe** [comment]
> Let me know what colors you want me to try and implement for the notification drop-downs

Maybe our brand colors over in https://github.com/github/old-primer/blob/master/scss/_variables.scss can give you a 💡 .

**Comment by brianamarie** [comment]
> I would be over the 🌔 for any help with .gif's especially

✋ I'm happy to help here. This may be a little different than with other courses because the activity is much more self-driven, and will look so different for every user. What kind of gifs are you envisioning for this course?

**Comment by hectorsector** [comment]
👋 @hollenberry! I wonder if we might leverage the ✨ work you've done here as a series. The content so far seems really well-suited for a topical webinar aligned with the [open source guides](https://github.com/github/open-source-guide), that we could then follow up with an on-demand offering. The on-demand offering could be fully activity driven which would align well with the other courses we have on the platform. This would allow learners the ability to engage at various levels (low entry barrier) and hit on the diverse learning styles (some folks learn best by watching a webinars, others like to read guides, and yet others want to jump right in to doing).

**Comment by hollenberry** [comment]
> I wonder if we might leverage the ✨ work you've done here as a series. The content so far seems really well-suited for a topical webinar aligned with the open source guides, that we could then follow up with an on-demand offering

@hectorsector I think that this is a great idea! I'd love to move in this direction. I have an idea that might be worth surfacing along these lines, which I'll draft up in an issue soon.

**Comment by hollenberry** [comment]
Closing this course out as we'll be presenting a new initiative to address a similar result.
